evap/staff/templates/staff_user_form.html

```diff
@@ -75,7 +75,37 @@ <h5 class="card-title me-auto">{% trans 'Export evaluation results' %}</h5>
         <div class="card card-submit-area text-center mb-3">
             <div class="card-body">
                 <button type="submit" class="btn btn-primary">{% trans 'Save user' %}</button>
+                {% if form.instance and form.instance.can_be_deleted_by_manager %}
+                                <button type="button" class="btn btn-danger btn-sm" onclick="deleteModalShow({{ form.instance.id }}, '{{ form.instance.full_name|escapejs }}');">
+                                    {% trans 'Delete user' %}
+                                </button>
+                            {% else %}
+                                <button type="button" disabled class="btn btn-sm btn-danger" data-toggle="tooltip" data-placement="left"
+                                    title="{% blocktrans %}This user contributes to an evaluation, participates in an evaluation whose participations 
+                                    haven't been archived yet or has special rights and as such cannot be deleted.{% endblocktrans %}">
+                                    {% trans 'Delete user' %}
+                                </button>
+                            {% endif %}
             </div>
         </div>
     </form>
 {% endblock %}
+
+{% block modals %}
+    {{ block.super }}
+    {% trans 'Delete user' as title %}
+    {% trans 'Do you really want to delete the user <strong id="data-label"></strong>?<br/>This person will also be removed from every other user having this person as a delegated or CC-user.' as question %}
+    {% trans 'Delete user' as action_text %}
+    {% include 'confirmation_modal.html' with modal_id='deleteModal' title=title question=question action_text=action_text btn_type='danger' %}
+    <script type="text/javascript">
+        function deleteModalAction(dataId) {
+            $.ajax({
+                type: "POST",
+                url: "{% url 'staff:user_delete' %}",
+                data: {"user_id": dataId},
+                success: function(){window.location="{% url 'staff:user_index' %}"},
```

Comment:
this is different than other form operations - usually, when submitting the form (or performing a different option that afterwards reloads the page) we want to show a message about the successful operation on the page. so in this case we need a message, that the user was successfully deleted.
so maybe we don't want to delete the user via an ajax request here? though this would require changes in the view as well. @He3lixxx what do you think?